dpctl/dptensor/numpy_usm_shared.py

```diff
@@ -271,13 +274,23 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         else:
             return NotImplemented
 
+    def __array_function__(self, func, types, args, kwargs):
+        fname = func.__name__
+        atypes = [type(x) for x in args]
+        has_func = isdef(fname)
+        dprint("__array_function__:", func, fname, type(func), types, atypes, has_func)
+        fargs = [x if not isinstance(x, ndarray) else np.ndarray(x.shape, x.dtype, x) for x in args]
```

Comment:
``np.ndarray(x.shape, x.dtype, x)`` is equivalent to ``x.view(np.ndarray)``, FYI.

```
In [1]: import dpctl

In [2]: import dpctl.dptensor.numpy_usm_shared as nps

In [9]: X = nps.ones(10)

In [10]: Y = X.view(np.ndarray)

In [11]: (type(X), type(Y))

Out[11]: (dpctl.dptensor.numpy_usm_shared.ndarray, numpy.ndarray)
```

Reply:
Thanks.  Your change of this in the code is fine.